about/about-network-policy.md

```diff
@@ -0,0 +1,223 @@
+than ever. 
+
+While you can (and should) use firewalls to restrict traffic at the perimeters of your network (commonly referred to as
+north-south traffic), their ability to police Kubernetes traffic is often limited to a granularity of the cluster as a
+whole, rather than to specific groups of pods, due to the dynamic nature of pod scheduling and pod IP addresses. In
+addition, the goal of most attackers once they gain a small foothold inside the perimeter is to move laterally (commonly
+referred to as east-west) to gain access to higher value targets, which perimeter based firewalls can't police against.
+
+Network policy on the other hand is designed for the dynamic nature of Kubernetes by following the standard Kubernetes
+paradigm of using abel selectors to define groups of pods, rather than IP addresses. And because network policy is
+enforced within the cluster itself it can police both north-south and east-west traffic.
+
+Network policy represents an important evolution of network security, not just because it handles the dynamic nature of
+modern microservices, but because it empowers dev and devops engineers to easily define network security themselves,
+rather than needing to learn low-level networking details or raise tickets with a separate team responsible for managing
+firewalls. Network policy makes it easy to define intent, such as "only this microservice gets to connect to the
+database", write that intent as code (typically in YAML files), and integrate authoring of network policies into git
+workflows and CI/CD processes.
+
+> Note: Calico and Calico Enterprise offer capabilities that can help perimeter firewalls integrate
+> more tightly with Kubernetes. However, this does not remove the need or value of network policies within the cluster itself.)
+{: .alert .alert-info }
+
+### Kubernetes network policy
+
+Kubernetes network policies are defined using the Kubernetes {% include open-new-window.html text='NetworkPolicy'
+url='https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#networkpolicy-v1-networking-k8s-io' %} resource. 
+
+The main features of Kubernetes network policies are:
+- Policies are namespace scoped (i.e. you create them within the context of a specific namespace just like, for example, pods)
+- Policies are applied to pods using label selectors
+- Policy rules can specify the traffic that is allowed to/from other pods, namespaces, or CIDRs
+- Policy rules can specify protocols (TCP, UDP, SCTP), named ports or port numbers
+
+Kubernetes itself does not enforce network policies, and instead delegates their enforcement to network plugins. Most
+network plugins implement the mainline elements of Kubernetes network policies, though not all implement every feature
+of the specification.  (Calico does implement every feature, and was the original reference implementation of Kubernetes
```

Comment:
{{site.prodname}}

Reply:
Can't use prodname for this, since Calico Enterprise was not the original reference implementation